Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```diff
@@ -1515,6 +1515,29 @@
   - type: Lathe
     materialUseMultiplier: 0.75
     timeMultiplier: 0.5
+    staticRecipes:
+    - BluespaceCrystal
```

Comment:
Remove this entire list from the T2 Ore Processor, and add every missing material to the Base Ore Processor.